docs/cel2/faq.md

```diff
@@ -100,10 +84,6 @@ Also see [Celo L2 Specification](https://specs.celo.org/root.html) for greater d
 
 See [What's changed section covering L1 fees](/cel2/whats-changed/op-l2#l1-fees).
 
-### I saw EigenDA mentioned, is it used?
-
-Yes! See [EigenDA](https://specs.celo.org/eigenda.html).
-
```

Comment:
Wondering why you removed this question?

Reply:
Essentially to shorten the FAQ and aim to only have questions in there that aren't easily addressed by a docs search. EigenDA is mentioned on https://docs.celo.org/cel2 (minor mention, but still) and https://docs.celo.org/cel2/whats-changed/cel2-architecture and there are many results when going to the docs internal search.